assets/src/domain/eventEditor/services/apollo/initialization/useI18nData.ts

```diff
@@ -1,7 +1,5 @@
-import { pathOr } from 'ramda';
-
 const useI18nData = (): any => {
-	return pathOr<any>({}, ['eeEditorData', 'i18n'], window);
+	return window?.eeEditorData?.i18n;
```

Comment:
in this case I see it was returning by default an empty object, we don't need that anymore?

Reply:
Oh yes, that's needed, thanks, good catch 👍